### DISCUSSION TITLE
What do you think about my Anchor extension?

### DISCUSSION THREAD

**Discussion by pftnhr** [datenstrom/community#894] [Ask a question] [answered]
@markseuffert made me build my very first extension [[1]](https://github.com/datenstrom/yellow/discussions/887#discussioncomment-6846569) and here it is: https://github.com/pftnhr/yellow-headinglink

It's not a big thing so be tough but fair, please.

**Comment by nevillepark** [comment]
Thank you for making this, I have been using [anchor.js](https://www.bryanbraun.com/anchorjs/) for this but would much prefer an extension. I will give it a try!

**Reply by pftnhr** [reply]
Thank you very much! Let me know if anything is still stuck or missing and I'll see if I can get it fixed.

**Comment by annaesvensson** [comment]
Good work. Maybe rename it to Anchor extension? Of course, it's up to you to pick a good name. From a user's point of view technical details are often not that important. Users probably look at a list of extensions and foremost check to see if there's anything on that list they can use to solve a problem. Short extension names are probably advisable, for example "Share" instead of "Minisocialshare". Descriptive extension names are probably advisable, for example "Edit" instead of "Webinterface". What sometimes could make sense are extension names that follow a familiar pattern, that's why we recommend to use city names for themes, for example "Stockholm". By the way, a big Swedish furniture store used to name front door mats after Danish cities, in case you didn't know there is a historical rivalry between Scandinavian countries, keep this in mind the next time you go [shopping](https://www.ikea.com/ie/en/cat/door-mats-10698/).

**Reply by pftnhr** [reply]
Technically, links are nothing more than anchors. I'll have to think about maybe renaming it Deepanchor, although I think Headinglink is reasonably self-explanatory.

**Comment by pftnhr** [comment, marked as answer]
@markseuffert has sent me a few changes that make the extension even easier to use. At the same time, we also renamed the extension to **_yellow-anchor_**, as we thought that might be a more appropriate name after all.

You can still find it at the link above.
